app/javascript/js/home/search/embedded_app/components/SearchBox.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {
+  forwardRef, useEffect, useImperativeHandle, useMemo, useRef, useState,
+} from 'preact/compat';
+import { delay } from 'lodash';
+import { debounceDelay } from 'js/home/search/embedded_app/helpers/SearchBoxHelper';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const searchIcon = require('../../../../../images/home/search.svg');
```

Comment:
Can't yo do this with modern imports?

Reply:
I tried, but did not work. No clue why. I try again, maybe it now works magically

Reply:
"Cannot find module '../../../../../images/home/search.svg' or its corresponding type declarations"